technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -0,0 +1,493 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" initialize-with="." default-locale="de-DE" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Technische Universität Dresden - Historische Musikwissenschaft (note)</title>
+    <title-short>TUD Historische Musikwissenschaft (note)</title-short>
+    <id>http://www.zotero.org/styles/technische-universitat-dresden-historische-musikwissenschaft-note</id>
+    <link rel="self" href="http://www.zotero.org/styles/technische-universitat-dresden-historische-musikwissenschaft-note"/>
+    <link href="http://www.zotero.org/styles/foerster-geisteswissenschaft" rel="template"/>
+    <link href="https://bildungsportal.sachsen.de/opal/auth/RepositoryEntry/7960428545/CourseNode/92427926399928/Merkblatt_Literaturangaben_historische_Musikwissenschaft.pdf" rel="documentation"/>
+    <author>
+      <name>Jens Mittelbach</name>
+      <uri>http://www.slub-dresden.de</uri>
+      <email>jens.mittelbach@slub-dresden.de</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <category field="generic-base"/>
```

Comment:
Can you remove the "generic-base" field line? We typically only use it when we can't assign any other field, or if a style clearly is meant for a multidisciplinary audience (like APA).